sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/coverage/internal/DefaultCoverage.java

```diff
@@ -76,6 +76,9 @@ public CoverageType type() {
 
   @Override
   public NewCoverage lineHits(int line, int hits) {
+    Preconditions.checkNotNull(inputFile, "Call onFile() first");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Call onFile() first" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1192)